starter-code/index.html

```diff
@@ -10,5 +10,33 @@
   <title>Ironhack cart</title>
 </head>
 <body>
+
+  
+<!-- TESTING CODE -->
+  <div class="main">
+    <h1 class="heading-text">Ironhack<br>Merchandise Shop</h1>
+    <div class="item">
+      <span class="product">IronBubble-head</span>
+      <span class="individualPrice" type="number">25</span>
+      <span>QTY</span>
+      <input class="input" type="number">
```

Comment:
Little tip, if you don't want to allow negative values, do:
```
<input type="number" min="0" />
```
Keep in mind this is not a perfect solution.